CHANGELOG.md

```diff
@@ -1,23 +1,13 @@
-# NEXT RELEASE
+# 5.7.2 Release notes
 
 ### Bugfixes
 
 * Fix a use-after-free when an observer is passed to rollback_and_continue_as_read().
```

Comment:
@tgoyne When could this happen? How would users see observe it? When was it introduced? What's the issue number?

Reply:
It has existed for as long as that functionality has been present (several years), and there is no issue for it.

Reply:
Ok, but how could users experience this?